kotlin/add-features-feature-service/README.md

```diff
@@ -1,33 +1,34 @@
-# Add features (Feature Service)
+# Add features
```

Comment:
I think this is an example of the scenario Jon mentioned earlier and then what Lucas clarified: let's leave this title as it was "Add features (Feature Service)" and we can catch any changes after the next Readme Title effort.